numba_dppy/tests/_helper.py

```diff
@@ -88,15 +88,15 @@ def is_windows():
 skip_windows = pytest.mark.skipif(is_windows(), reason="Skip on Windows")
 
 skip_no_opencl_gpu = pytest.mark.skipif(
-    not has_opencl_gpu(),
+    not has_opencl_gpu() and config.TESTING_SKIP_NO_OPENCL_GPU,
```

Comment:
`has_opencl_gpu()` is supposed to skip a test when there is no opencl GPU already, so why do we need this? Not clear from the PR description.